src/identityApiClient.interfaces.ts

```diff
@@ -1,44 +0,0 @@
-import { IdentityApiData, MPID, UserIdentities } from '@mparticle/web-sdk';
```

Comment:
for some reason i thought we were goin to keep the `.interfaces.ts` files to keep each module smaller and to ensure separation of interfaces from code.  if they were temporary because we are migrating, i'm fine with that.  i just want to make sure we are picking a path and sticking with it

Reply:
I think my intention was to always have the `.interfaces.ts` as temporary until we migrate completely to TypeScript.

Other than intentionally mirroring native (which splits implementation and interfaces), I think the only reason to keep them separate is if the module is too large or complex and having a separate file makes it easier to compare the interface and implementation. However, in that case, I would opt to refactor or break up the module.

Reply:
Chatted on Zoom and decided that we should remove the `.interface.ts` file after migrating a module, unless the module is longer than 500 lines of code.